topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+>
+> a) There seems to be **no horizontal gene transfer** (HGT) in the MTBC. HGT complicates phylogenetic inference in many bacteria because a piece of DNA introduced by HGT has a different history and thus phylogeny than genes not affected by HGT.
+>
+> b) There is **little genetic diversity** in the MTBC: any two strains differ by only around 2,500 SNPs over the whole genome. To achieve a good resolution of recent evolution (e.g. during an outbreak), whole genome sequences are thus extremely useful. With whole genome sequences, resolution can be further increased by not only considering SNPs, but also other types of mutations, in particular short insertions and deletions (as e.g. shown in [this study]( https://doi.org/10.1371/journal.ppat.1008357)).
+>
+> c) A large proportion of DNA polymorphisms in the MTBC are **singletons**, that is, variants present only in a single strain. This adds to the problem of low diversity, since singletons are not informative about tree topology.
+>
+> d) In this workshop, and indeed in many studies of the MTBC, SNPs are called not against H37Rv, but against a reconstructed ancestral genome. This means that the number of SNPs identified does not reflect the evolutionary distance from some random strain like H37Rv, but from the most recent common ancestor of the MTBC. Below we will see that this has implications for the interpretation of a tree.
+>
+{: .comment}
+
+
+# The alignment
+Aligned DNA or protein sequences are the starting material for phylogenetic inference with molecular data. Here we will make use of the single nucleotide polymorphisms (SNPs) you obtained in the previous tutorials on SNP calling and transmission clusters, reflecting the diversity of 19 MTBC strains and 1 strain of **M. canettii**. The latter is included to have an outgroup, allowing us to root the phylogeny.
+
+Recall that the alignment here was generated using SNPs called from reference-aligned short reads. A frequently used alternative approach to obtain a phylogeny from short read data is to a) assemble the genomes (see the numerous [Galaxy tutorials](https://training.galaxyproject.org/training-material/topics/assembly) on this topic), b) annotate genes, c) extract genes present in all strains (the "core" genes), d) align the core genes. This approach underlies core genome multilocus sequence typing (cgMLST), which is often used to genotype bacterial pathogens (e.g. [Zhou et al. 2021](https://doi.org/10.1093/bioinformatics/btab234)).
+
+An alignment of SNPs looks something like this. Each row is a different strain, each column a position in the reference genome. Because this alignment is based on SNPs, it contains only variable positions. Inspecting the alignment is always a good idea, because also in phylogenetics the principle of 'garbage in, garbage out' applies.
+
+<img src="./images/MEGA_alignment.png" alt="drawing" width="500"/>
+<br/><br/>
```

Comment:
```suggestion
![drawing](./images/MEGA_alignment.png){:width="500"}

Reply:
Why this alternative image formatting here and not for the other images?

Reply:
No, exactly, please do it for all images! I just didn't feel like making the changes for all of them, so this is just an example, please replicate it for the others :)